master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/index.md

```diff
@@ -22,25 +25,80 @@ You must have a cluster which meets the following requirements:
 - Your Kubernetes controller manager is configured to allocate pod CIDRs (i.e. by passing `--allocate-node-cidrs=true` to the controller manager)
 - Your Kubernetes controller manager has been provided a cluster-cidr (i.e. by passing `--cluster-cidr=192.168.0.0/16`, which the manifest expects by default).
 
+> Note:  If you are upgrading from Calico v2.1, the cluster-cidr selected for your controller manager should remain
+> unchanged from the v2.1 install (the v2.1 manifests default to `10.244.0.0/16`).
+
 ## Installation
 
-To install Calico, ensure you have a cluster which meets the above requirements and run one of the following commands based on your Kubernetes version:
+This document describes three installation options for the two operational modes
+for Calico (Calico with three installation options:
```

Comment:
Looks like a half edit.  You can probably just get rid of `(Calico with three installation options`